app/questionnaire/placeholder_parser.py

```diff
@@ -58,6 +63,9 @@ def __init__(
         self._placeholder_map: MutableMapping[
             str, Union[ValueSourceEscapedTypes, ValueSourceTypes, None]
         ] = {}
+        self._routing_path_block_ids = routing_path_block_ids
```

Comment:
Do we need to create all these private vars when in reality they are not being used except as attributes for the resolver? We only really need transformer and placeholder_map:
<img width="636" alt="Screenshot 2023-02-21 at 10 22 04" src="https://user-images.githubusercontent.com/53475968/220318233-87009ddd-cc04-4a21-849f-bdd263d5ee9a.png">

Reply:
Updated this